.goreleaser.yml

```diff
@@ -183,24 +174,51 @@ dockers:
   extra_files:
     - "images/hybrid/"
 
-# Build plugin-compiler multiarch
+
+# Build plugin-compiler
 - ids:
     - std
   image_templates:
-    - "tykio/tyk-plugin-compiler:{{ .Tag }}"
+    - tykio/tyk-plugin-compiler:{{ .Tag }}
     - "tykio/tyk-plugin-compiler:v{{ .Major }}.{{ .Minor }}{{.Prerelease}}"
   build_flag_templates:
     - "--label=org.opencontainers.image.created={{.Date}}"
     - "--label=org.opencontainers.image.title=tyk-plugin-compiler"
     - "--label=org.opencontainers.image.revision={{.FullCommit}}"
     - "--label=org.opencontainers.image.version={{.Version}}"
     - "--build-arg=TYK_GW_TAG={{ .Tag }}"
+    - "--build-arg=GOLANG_CROSS={{ .Env.GOLANG_CROSS }}"
   goarch: amd64
   goos: linux
   dockerfile: images/plugin-compiler/Dockerfile
   extra_files:
```

Comment:
Why we need this section here, if we copy tar.gz? 
Having such a long list of files and dirs can cause a lot of human errors.

Reply:
This way we don't have a tar.gz making local builds simpler. And we'll have to maintain this list, either implicitly in a tarball or explicitly here.